backend/apps/owasp/graphql/queries/__init__.py

```
@@ -4,8 +4,9 @@
 from .committee import CommitteeQuery
 from .event import EventQuery
 from .project import ProjectQuery
+from .snapshot import SnapshotQuery
 from .stats import StatsQuery
 
 
-class OwaspQuery(ChapterQuery, CommitteeQuery, EventQuery, ProjectQuery, StatsQuery):
+class OwaspQuery(ChapterQuery, CommitteeQuery, EventQuery, ProjectQuery, SnapshotQuery, StatsQuery):
```

Ruff (0.8.2)

Line too long (100 > 99)

(E501)